### PR TITLE
Add console template for Prometheus itself

### PR DESCRIPTION
**Pull request by brian-brazil**
@beorn7

### CODE REVIEW

File: consoles/index.html.example [hunk 1]
```diff
@@ -18,6 +18,14 @@
   <td><code>node</code></td>
 </tr>
 <tr>
+  <td>Prometheus</td>
+  <td><code>prometheus</code></td>
+</tr>
+<tr>
+  <td>SNMP Exporter</td>
```
Comment (juliusv) [inline consoles/index.html.example]: This was forgotten earlier, ja?

Comment (fabxc) [inline consoles/index.html.example]: Yes

**Comment by juliusv** [comment]
It could also be interesting to graph the rate of incoming expression queries. Something like:

```
rate(http_requests_total{handler=~"^query",job="prometheus"}[5m])
```

File: consoles/index.html.example (hunk 1 above)
Comment (brian-brazil) [inline consoles/index.html.example]: Ja

**Comment by brian-brazil** [comment]
@juliusv added

**Comment by juliusv** [comment]
Cool! :+1:

**Pull request merged**